src/App.js

```diff
@@ -1,26 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
+import App from './app';
```

Comment:
This should be changed to './app/App' else some import error would be seen

Reply:
There is an index.jsx file in app